### ISSUE TITLE
Issues with displaying video [solved]

### ISSUE THREAD

**Issue by 4325123** [rogro82/PiPup#24]
I'm trying to figure out what I can do in order to display video.  My cameras require authentication through the URL (http://u:p@ip), but I've read that PiPup doesn't support it.  I do have an rtsp stream that doesn't require authentication.  I've tried to take my rtsp stream and transcode it with vlc with this command:

$ vlc': vlc -R rtsp://10.0.110.2:8554/proxyStream-1 --sout "#transcode{vcodec=mjpg,vb=2500,scale=1.0,fps=10,acodec=none}:standard{access=http{mime=multipart/x-mixed-replace; boundary=7b3cc56e5f51db803f790dad720ed50a},mux=mpjpeg,dst=:8888/videostream.cgi}"

I can open it in another vlc client, but it doesn't do anything when trying to open it with PiPup.  

I'm trying to figure out where to go from here with troubleshooting.  If I understand this correctly, I need to have the stream open in the webview app?  Is there additional logs somewhere to try to figure out why it's failing?

The jpeg's work great and am happy I stumbled on this project.  Thanks for the hard work in getting this done!!!

**Comment by 4325123** [comment]
OK, whenever I post, I figure it out shortly later.  Here's what I did in case somebody else needs it.  

I have a security camera that streams rtsp and mjpeg but require authentication.  I ended up using a live555proxyserver to proxy the rtsp streams.  I did this so I can get rid of authentication, so it runs faster, and I can have multiple workstations/devices grab the stream and not hang up the camera.  I've had this setup way before PiPup.

I then discovered streameye.  Once I got that compiled and running, I can execute:

$ ffmpeg -i rtsp://10.0.110.2:8554/proxyStream-2 -f mjpeg - 2>/dev/null | ./streameye -d

Once that is running, I can use pipup to trigger on this...

        "media": { "web": { "uri": "http://10.0.110.2:8080"}}

And boom....I now have live video showing in the notification.  

Thank you again for creating this.  With a days time of work, I was able to get this required feature up and running on my android tv.
